website_sale_hide_price/models/product_template.py

```diff
@@ -38,3 +38,18 @@ def _get_combination_info(
             }
         )
         return combination_info
+
+    def _search_render_results(self, fetch_fields, mapping, icon, limit):
+        res = super()._search_render_results(fetch_fields, mapping, icon, limit)
+        new_result = []
+        for data in res:
+            product = self.env["product.template"].browse(data.get("id"))
```

Comment:
It looks like in the original method you can rely on zipping de results along with the recordset so you don't have to browse it again (this will underperform over large amounts of records):

https://github.com/odoo/odoo/blob/fa525cbcf63c4e35f224f6df41e9bc9fe356c7b7/addons/website_sale/models/product_template.py#L391

Reply:
Done!, thank you